recipes/r-diptest/meta.yaml

```diff
@@ -51,4 +51,4 @@ about:
 
 extra:
   recipe-maintainers:
-    ArneKr
+    - ArneKr
```

Comment:
Argh! The linter should catch that :unamused: 

Thanks @isuruf for catching and fixing it!